Feat : DIG-85 피드 운동일지 추가, 삭제 구현

Dongjin113 opened the pull request:
### 피드 운동일지 삭제 구현
- [x] 피드 운동일지 삭제시 피드 이미지 삭제
- [x] 피드 운동일지 삭제시 피드 좋아요 삭제
- [x] 피드 운동일지 삭제시 피드 댓글 삭제
- [x] 피드 운동일지 삭제
- [x] 피드 운동일지 삭제후 운동일지 공개여부 변경

### 운동일지 공유시 피드 운동일지 생성
- [x] 운동일지 공유시 들어온 이미지 저장
- [x] 운동일지 공유시 피드운동일지 생성


---
### 운동완료한 운동일지 수 조회

### TimeTemplate 위치 변경

---
### 피드 운동일지 스크랩 추가

### ExerciseJournalRepository 버그 수정
- [ ] springDataJPA를 countByUserAndCompleted로 사용하여 빌드에러가 발생 -> countByUserAndIsCompleted로 변경

### 피드운동일지 삭제 후 공개여부 변경
- [ ] 메서드 이름 feedJournalDelete -> privateVisible 피드운동일지를 삭제하는 것보다 상태를 변경해주는 것이기 때문에 메서드 이름 변경

Dongjin113 committed:
Feat : DIG-85 피드 운동일지 추가, 삭제 구현

- [ ] 피드 운동일지 삭제시 피드 이미지 삭제
- [ ] 피드 운동일지 삭제시 피드 좋아요 삭제
- [ ] 피드 운동일지 삭제시 피드 댓글 삭제
- [ ] 피드 운동일지 삭제
- [ ] 피드 운동일지 삭제후 운동일지 공개여부 변경

- [ ] 운동일지 공유시 들어온 이미지 저장
- [ ] 운동일지 공유시 피드운동일지 생성

dunowljj commented on src/main/java/com/ogjg/daitgym/domain/feed/FeedExerciseJournal.java:
```diff
@@ -24,4 +27,10 @@ public class FeedExerciseJournal extends BaseEntity {
     private ExerciseJournal exerciseJournal;
 
+    @OneToMany(mappedBy = "feedExerciseJournal")
+    private List<FeedExerciseJournalImage> feedExerciseJournalImage = new ArrayList<>();
```
컬렉션인데 단수형이네요! s 하나 부탁드려요!

Dongjin113 replied:
수정해숨다ㅣㅇ

Dongjin113 committed:
Feat : DIG-88 운동일지 수 조회 구현

### 운동완료한 운동일지 수 조회
### TimeTemplate 위치 변경

Dongjin113 committed:
Feat : DIG-89 피드 운동일지 스크랩 추가,
ExerciseJournalRepository 버그 수정

### 피드 운동일지 스크랩 추가

### ExerciseJournalRepository 버그 수정
- [ ] countByUserAndCompleted로 사용하여 빌드에러가 발생 -> countByUserAndIsCompleted로 변경

### 피드운동일지 삭제 후 공개여부 변경
- [ ] 메서드 이름 feedJournalDelete -> privateVisible 피드운동일지를 삭제하는 것보다 상태만 변경해주는 것이기 때문에 메서드 이름 변경
